### ISSUE TITLE
How to convert multiple strings to audio parallely?

### ISSUE THREAD

**Issue by tshrinivasan** [tshrinivasan/tamil-tts-install#1]
The tamil TTS from IITM donlab and SSN College of Engineering can process one string at a time and store it as wav/1.wav always.

Because of this we can not process multiple files at same time.

The current tts gets a string as input.
The output is stored as ssn_hts_demo/wav/1.wav

The script that generates is ssn_hts_demo/scripts/complete

Explore the perl script and make sure that it stores the output as wav/book_name/line_no.wav

so that we can run the script in parallel to process multiple books.

**Comment by tshrinivasan** [comment]
The new pull request by mohan solved this
